api/loaders/transactions/index.js

```diff
@@ -1,6 +1,35 @@
+import _ from 'lodash';
+
 import {getTimelineInfo, getTimelinePage} from '../timeline';
 import {parseTransaction} from '../parseMessage';
 
+/** see inbound-stream.js#process() -> (message.t === 'entry') */
+async function loadEntryInfo(redisX, entryId) {
+  return await redisX.hgetallAsync([`!ent:${entryId}`]);
+}
+
+// augment with block_id field since it's not available in the timeline
+async function fixupTimelinePage(redisX, timelinePage) {
```

Comment:
`block_id` is a required UI element so we need to fixup transactions with that data